utils/enphase.js

```diff
@@ -4,9 +4,30 @@ const moment = require('moment');
 const baseUrl = 'https://api.enphaseenergy.com/api/v2';
 const key = process.env.ENPHASE_KEY;
 
+const EnphaseSettings = {
+  rechN8WXaM60Xx859: data => data.meter_production, // lora foo Unit B
+  recMzaIHIl5Uq1xRJ: data =>
+    data.micro_production.map((v, i) => v - data.meter_production[i]) // lora foo unit A
```

Comment:
would this mean that when a new user is onboarded, we would need to update this object?

Reply:
Yup. This was what was brought up in conversation with Grayson: until we can find a better way to query per-module, we’ll have to do it this way. Once we handle errors better, this will be an error that is brought up if you a subscriber doesn’t have something here